File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseVenue.java
package net.squanchy.service.firebase.model;

import android.support.annotation.NonNull;

public class FirebaseVenue {

    @NonNull
    public String name;

    @NonNull
    public String address;

    public double lat;

    public double lon;

    @NonNull
    public String description;
}

File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseVenue.kt
package net.squanchy.service.firebase.model

data class FirebaseVenue(
        var name: String? = null,
        var address: String? = null,
        var lat: Double? = null,
        var lon: Double? = null,
        var description: String? = null,
        var map_url: String? = null,
        var timezone: String? = null
)
